content/articles/how-to-create-an-android-wear-os-app/index.md

```diff
@@ -0,0 +1,269 @@
+    xmlns:tools="https://schemas.android.com/tools"
+    android:layout_width="409dp"
+    android:layout_height="wrap_content"
+    android:padding="8dp" >
+  
+      <TextView
+          android:id="@+id/text_1"
+          android:layout_height="wrap_content"
+          android:layout_width="match_parent"
+          android:text="Hello world" />
+  
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp">
+
+        <android.support.wear.widget.SwipeDismissFrameLayout
+            android:layout_width="match_parent"
+            android:id="@+id/slide_dissmiss"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/test_content"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                android:text="Slide"/>
+        </android.support.wear.widget.SwipeDismissFrameLayout>
+    </FrameLayout>
+</android.support.wear.widget.BoxInsetLayout>
+```
+
+### Navigation
+This refers to starting or opening new destinations from the other.
+
+#### Using NavigationDrawer
+The watch application normally does not have a title bar to conserve significant display space. When using ViewPager, TabLayout is not displayed, but we can't be sure what page we're on.
+
+Let's have a look at the effect of the navigation bar `WearableNavigationDrawerView`, which is utilized to fix this problem.
+
+A navigation bar will display as you slide down from the top of the screen, revealing the icon and title of the current page. If there are numerous pages, slide the page side to side to change it.
+
+Let's look at how it's used. We change the layout file and make WearableDrawerLayout the root layout, as well as adding a navigation bar control.
+
+```xml
+<android.support.wear.widget.drawer.WearableNavigationDrawerView
+        android:id="@+id/navigation_drawer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:navigationStyle="multiPage"/>
+```
+
+Set an adapter for the ViewPager, as well as the icon and title for each page, as shown below.
+
+```kotlin
+class DrawerAdapter(context: Context) : WearableNavigationDrawerView.WearableNavigationDrawerAdapter() {
+
+    private val context = context
+
+    override fun getItemText(pos: Int): CharSequence {
+        return when (pos) {
+            0 -> "First page"
+            else -> "The second page"
+        }
+    }
+
+    override fun getItemDrawable(pos: Int): Drawable {
+        return when (pos) {
+            0 -> ContextCompat.getDrawable(context, R.drawable.icon_one)!!
+            else -> ContextCompat.getDrawable(context, R.drawable.icon_two)!!
+        }
+    }
+
+    override fun getCount(): Int {
+        return 2
+    }
+
+}
+```
+
+Set it in the Activity.
+
+```kotlin
+drawer.setAdapter(MainDrawerAdapter(this))
```

Comment:
You might have forgotten to use the correct ID. Please confirm and reference it properly. Otherwise, this won't work.